lib/puppet/util/plist.rb

```diff
@@ -60,7 +61,7 @@ def parse_plist(plist_data, file_path = '')
 
       begin
         plist_obj = new_cfpropertylist(:data => plist_data)
-      rescue CFFormatError => e
+      rescue CFFormatError, NoMethodError => e
```

Comment:
ACTUALLY now that I think about it more, does it make more sense for us to rescue `NoMethodError` in a separate block so we can format something to the extent of "This is probably invalid plist data" in the log message?